packages/opentelemetry-test-utils/src/instrumentations/index.ts

```diff
@@ -38,17 +41,27 @@ export const mochaHooks = {
         // could not determine serviceName, continue regardless of this
       }
     }
-    const provider = registerInstrumentationTestingProvider({
-      resource: new Resource({
-        [SemanticResourceAttributes.SERVICE_NAME]: serviceName,
-      }),
+    const testResource = new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: serviceName,
     });
-    getInstrumentation()?.setTracerProvider(provider);
+
+    const providers = registerInstrumentationTestingProvider(
+      {
+        resource: testResource,
+      },
+      {
+        resource: testResource,
```

Comment:
This seems superfluous to repeat the identical options. Can we try to merge the trace provider options and the meter provider options? As a reference: https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-sdk-node/src/types.ts#L29.

Reply:
Sorry for the delay, fixed that comment :)